Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN dotnet publish -r linux-musl-x64 -c Release -o /opt/test-runner --no-restore
 RUN dotnet add package Microsoft.NET.Test.Sdk -v 16.8.3 && \
     dotnet add package xunit -v 2.4.1 && \
     dotnet add package xunit.runner.visualstudio -v 2.4.3 && \
-    dotnet add package FsUnit.xUnit -v 4.0.4
+    dotnet add package FsUnit.xUnit -v 4.0.4 && \
+    dotnet add package Exercism.Tests -v 0.1.0-alpha --prerelease
```

Comment:
Is this correct or do you want to release that first?

Reply:
It's fine for now. There is no real disadvantage to having it be alpha and I want to test out some things.